apps/docs/src/app/platform/platform-documentation.module.ts

```diff
@@ -127,7 +133,14 @@ import { ScrollingModule } from '@angular/cdk/scrolling';
         PlatformRadioGroupListExampleComponent,
         PlatformRadioGroupContentExampleComponent,
         PlatformRadioGroupDisabledExampleComponent,
-        PlatformRadioGroupDocsComponent
+        PlatformRadioGroupDocsComponent,
```

Comment:
Could you guys consider creating lazy loaded modules?
Pulling all this code at once affects our documentation page usage experience.

Reply:
make sense. not only docs but also tests.

Reply:
@fkolar @salarenko this change will be taken in a diffrent PR